build.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project default="xar" name="usaybia">
+<project default="xar" name="srophe">
     <xmlproperty file="expath-pkg.xml"/>
     <property name="project.version" value="${package(version)}"/>
-    <property name="project.app" value="usaybia"/>
+    <property name="project.app" value="srophe"/>
```

Comment:
Does this need to be kept as "usaybia" or "usaybia.net" or will it function okay like this?